src/main.py

```diff
@@ -21,15 +21,15 @@
 import argparse
 from collections.abc import Iterable
 from typing import Union
+from BenchmarkManager import BenchmarkManager
 
 install_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 sys.path.append(install_dir)
-from BenchmarkManager import BenchmarkManager
```

Comment:
The import of BenchmarkManager should happen after adding the path , so I would suggest the following order of imports:
import sys
import os
install_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
sys.path.append(install_dir)
import argparse
from collections.abc import Iterable
from typing import Union
from BenchmarkManager import BenchmarkManager